Modules/_testcapimodule.c

```diff
@@ -5058,7 +5058,7 @@ static PyMethodDef TestMethods[] = {
     {"get_mapping_items", get_mapping_items, METH_O},
     {"test_pythread_tss_key_state", test_pythread_tss_key_state, METH_VARARGS},
     {"hamt", new_hamt, METH_NOARGS},
-    {"bad_get", bad_get, METH_FASTCALL},
+    {"bad_get", bad_get, METH_VARARGS},
```

Comment:
I don't see the point of making this change on a single function. There are dozens of functions implemented with METH_FASTCALL which emit the same warning.

Why not using (PyCFunction)(void*) as done in bpo-33012? I suggest to fix all warnings at once in bpo-33012.
https://bugs.python.org/issue25750#msg328539

Reply:
Functions implemented with METH_FASTCALL emit different warning. All functions in this module use public `METH_*`. I'm not sure that using `(PyCFunction)(void*)` is the best way. If there is no necessary to use METH_FASTCALL for this function, I prefer to use METH_VARARGS. This will allow us to avoid any casting at all.

Reply:
> If there is no necessary to use METH_FASTCALL for this function

The whole point of that test goes away when you use `METH_VARARGS`. It tests a crash which occurred only with `METH_FASTCALL`.

Reply:
Thank you. Merged changes use METH_FASTCALL.